src/tools/wasm-emscripten-finalize.cpp

```diff
@@ -135,6 +137,11 @@ int main(int argc, const char* argv[]) {
          [&checkStackOverflow](Options* o, const std::string&) {
            checkStackOverflow = true;
          })
+    .add("--wasi",
+         "",
+         "Emit a wasi-compatible wasm file, with exported_start etc.",
```

Comment:
exported_start should be exported _start maybe?

Reply:
Oops, yes, that's a typo, thanks! Fixing now.